### PR TITLE
easy_bar_mean + easy_bar_median + easy_range

### PR DESCRIPTION
**Pull request by feddelegrand7**
Hi, I've tested the functions using vdiffr. Feel free to edit or ignore this pull request.

### CODE REVIEW

File: tests/testthat/test-easy_bar.R [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    stat_summary(fun.data = summary,geom = "errorbar",  colour = "brown1", size = 2) +
+    xlab("")+
+    ylab("range")
+
+  vdiffr::expect_doppelganger("easy range", easy_range)
+  vdiffr::expect_doppelganger("ggplot2 range", ggplot_range)
+
+})
+
+install.packages("patchwork")
```
Comment (jonocarroll) [inline tests/testthat/test-easy_bar.R:71]: This is the cause of the CI test failure.

**Comment by jonocarroll** [comment]
Thank you! 

To be honest I'm not sure we want to try to wrap the entire plotting code. The other functions in the package are mainly wrappers to `theme()` additions.

More critically, I'm not sure that a bar plot is an appropriate way to represent mean or median at all. I see you have a mean, median, and errorbar version, but I'd instead use a boxplot to expose the range of the data. I've added the mean to this here but I wouldn't usually use that statistic

``` r
library(ggplot2)
ggplot(mtcars, aes(factor(am), mpg)) + geom_boxplot() + stat_summary(fun = mean)
#> Warning: Removed 2 rows containing missing values (geom_segment).
```



<sup>Created on 2020-04-12 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

I do appreciate the PR though, thanks again.

**Comment by feddelegrand7** [comment]
Hi, thanks for the feedback. That's ok I understand. The cool thing is that I've discovered the vdiffr package which I didn't heard about before. Really good experience. Keep up !

**Comment by feddelegrand7** [comment]
Yeah I totally agree with the box plot aspect but the problem of box plot is that many people are not familiar with them which may create some friction, but yes box plot indeed provide all the necessary info at once.

**Pull request closed without merging**